src/runtime/execution.rs

```diff
@@ -227,11 +227,12 @@ impl ExecutionState {
     where
         F: FnOnce(&mut ExecutionState) -> T,
     {
-        EXECUTION_STATE.with(|cell| f(&mut *cell.borrow_mut()))
+        Self::try_with(f).expect("Shuttle internal error: cannot access ExecutionState. are you trying to access a Shuttle primitive from outside a Shuttle test?")
```

Comment:
I'm late to the party for this nit, but `are` at the beginning of the second sentence should be capitalized.